Core/Inc/analyzer.h

```diff
@@ -33,4 +34,14 @@ AccumulatorData_t* bmsdata;
 
 AccumulatorData_t* prevbmsdata;
 
+/**
+ * @brief Create a new object to store the most recent data point for the temp sensor
+ */
+sht30_t* sht30data;
```

Comment:
seems like the @brief got doubled here, but also dont think we need one at all. creating an sht30 object right nect to an acc data one is pretty self explanatory imo, so feel free to remove both